style/pi-hole.css

```diff
@@ -310,4 +310,13 @@ code.breakall
 .bootstrap-select.bs-container.align-right {
     left: unset !important;
     right: 10px;
+}
+
+.warning {
+    color: orange;
+    font-size: 18px;
+}
+
+.hidden {
```

Comment:
I'd rename this to `d-none` and put a reminder to re-use the class in the whole codebase instead of inline CSS or JS code.

Reply:
@DL6ER you missed adding the needed CSS for `d-none` :)